worker/rhpv3.go

```diff
@@ -1133,39 +1137,62 @@ func RPCAppendSector(ctx context.Context, t *transportV3, renterKey types.Privat
 		return
 	}
 
+	// compute expected collateral and refund
+	expectedCost, expectedCollateral, expectedRefund, err := uploadSectorCost(pt, rev.WindowEnd)
+	if err != nil {
+		return types.Hash256{}, types.ZeroCurrency, err
+	}
+
+	// apply leeways.
+	// TODO: remove once most hosts use hostd. Then we can check for exact values.
+	expectedCollateral = expectedCollateral.Mul64(9).Div64(10)
+	expectedCost = expectedCost.Mul64(11).Div64(10)
+	expectedRefund = expectedRefund.Mul64(9).Div64(10)
+
+	// check if the cost, collateral and refund match our expectation.
+	if executeResp.TotalCost.Cmp(expectedCost) > 0 {
+		return types.Hash256{}, types.ZeroCurrency, fmt.Errorf("cost exceeds expectation: %v < %v", executeResp.TotalCost.String(), expectedCost.String())
```

Comment:
can you flip these in the error message: "cost exceeds expectation: %v > %v" makes more sense?